apps/state_mediator/test/state_mediator/integration/gtfs_test.exs

```diff
@@ -163,33 +163,6 @@ defmodule StateMediator.Integration.GtfsTest do
       assert invalid_services == []
     end
 
-    test "keeps Winsor Gardens and Plimptonville in the right order" do
```

Comment:
Plimptonville no longer has service and will no longer, even when we begin to add back service in the future.